### DISCUSSION TITLE
Cannot find reference 'Tortoise' in 'imported module tortoise'

### DISCUSSION THREAD

**Discussion by gaurishhs** [tortoise/tortoise-orm#1229] [Q&A]
![Screenshot 2022-08-25 at 5 08 05 PM](https://user-images.githubusercontent.com/78847111/186654471-40ba3474-b92a-4077-85d7-a897951548b8.png)
![Screenshot 2022-08-25 at 5 08 57 PM](https://user-images.githubusercontent.com/78847111/186654647-fc673a7d-e53d-47c4-a7cd-33430db5aec8.png)

**Comment by Chise1** [comment]
You check to see if you already have a package(named tortoise).
